commands/game.js

```diff
@@ -0,0 +1,101 @@
+const { SlashCommandBuilder, ActionRowBuilder, ButtonBuilder, ButtonStyle, EmbedBuilder, Event, IntegrationApplication } = require('discord.js');
+const fs = require("fs");
+const util = require("util");
+const { createLogger, format, transports } = require('winston');
+
+const logger = createLogger({
+    level: `info`,
+    format: format.json(),
+    defaultMeta: { service: `game` },
+    transports: [
+      //
+      // - Write to all logs with level `info` and below to `console.log` 
+      // - Write all logs error (and below) to `error.log`.
+      //
+      new transports.File({ filename: `error.log`, level: `error` }),
+      new transports.File({ filename: `console.log` }),
+      new transports.Console()
+    ]
+  });
+
+const leaderboard = createLogger({
+    level: `info`,
+    format: format.json(),
+    defaultMeta: { service: `leaderboard` },
+    transports: [
+      new transports.File({ filename: `leaderboard.log` }),
+      new transports.Console()
+    ]
+});
+
+const readFile = util.promisify(fs.readFile);
+
+module.exports = {
+	data: new SlashCommandBuilder()
+		.setName(`game`)
+		.setDescription(`Let's play a game!`),
+	async execute(interaction) {
+        const data = await readFile("questions.json");
```

Comment:
This duplicates a ton of `quiz.js` logic, so we should eventually break it out and make those components reusable, but it's whatevs for now

Reply:
Agreed. We should prioritize correcting the DiscordAPIError[40060]: Interaction has already been acknowledged. 

Next Sprint!